hc/src/help2.src

```diff
@@ -2341,6 +2350,14 @@ author of "A Hypercomplex Calculus with Applications to Relativity"
 See also {Quaternion} and { Quaternion and Hypercomplex Algebra }
 ;
 ;
+~Topic=BURNINGSHIP, Label=HT_BURNINGSHIP
+(type=burningship)
```

Comment:
`burning-ship`

Reply:
See above comments

Reply:
Updated to:
~Topic=BURNING-SHIP, Label=HT_BURNING_SHIP
(type=burning-ship)